pages.ja/common/cat.md

```diff
@@ -0,0 +1,24 @@
+# cat
+
+> ファイルの出力と連結を行います。
+> 詳しくはこちら: <https://www.gnu.org/software/coreutils/cat>
+
+- ファイルの内容を標準出力に出力する:
+
+`cat {{ファイルパス}}`
+
+- 複数ファイルを連結して1つの出力ファイルにする:
+
+`cat {{ファイルパス1 ファイルパス2 ...}} > {{出力ファイルパス}}`
+
+- 複数ファイルを1つの出力ファイルに追加する:
+
+`cat {{ファイルパス1 ファイルパス2 ...}} >> {{出力ファイルパス}}`
+
+- ファイルの内容をバッファリング(一時保存)せずに出力ファイルにコピーする:
+
+`cat -u {{/dev/tty12}} > {{/dev/tty13}}`
```

Comment:
I considered translating "tty" into Japanese as "controlling terminal(制御端末)" but decided to leave it as is because I thought it would be more difficult to understand.

Reply:
Yeah, the current ones work. I will mark this conversation as resolved.